lxd/instance/drivers/driver_qemu.go

```diff
@@ -6351,14 +6351,14 @@ func (d *qemu) Info() instance.Info {
 		return data
 	}
 
-	hostArch, err := osarch.ArchitectureGetLocalID()
+	d.architecture, err = osarch.ArchitectureGetLocalID()
```

Comment:
hmm, but `d.architecture` is meant to be the instance's architecture, not the host's, no?

We don't currently support using the personality architecture for VMs (i686 on x86_64), but we may well enable it in the future, so it feels wrong to abuse d.architecture in this way.

Reply:
OK will revert

Reply:
Fixed